src/panel.js

```diff
@@ -116,6 +149,37 @@ const addLearnHowFBCWorksButton = async (fragment) => {
   addSpan(button, "sites-added-subhead");
 };
 
+const addCustomSiteButton = async (fragment, panelId) => {
+  const shouldShowRemoveSiteButton = await isSiteInContainer(panelId);
+  let button;
```

Comment:
We don't return this value from the function, so we can probably drop it?

```js
 const addCustomSiteButton = async (fragment, panelId) => {
   const shouldShowRemoveSiteButton = await isSiteInContainer(panelId);
   if (shouldShowRemoveSiteButton) {
     addFullWidthButton(fragment, "remove-site-from-container");
     addSpan(button, "button-remove-site");
     addTooltip(button, "button-remove-site-tooltip");
     return;
   }
   addFullWidthButton(fragment, "add-site-to-container");
   addSpan(button, "button-allow-site");
}
```

Reply:
The `button` is passed as an argument in both the `addSpan` and `addTooltip` functions. I think we need to keep it.